shell/common/engine.cc

```diff
@@ -279,6 +279,11 @@ void Engine::RunFromSource(const std::string& main,
   RunBundleAndSource(bundle_path, main, packages);
 }
 
+void Engine::SetAssetBundlePath(const std::string& bundle_path) {
+  TRACE_EVENT0("flutter", "Engine::SetAssetBundlePath");
```

Comment:
Should this be `TRACE_EVENT_INSTANT0`?

Reply:
TRACE_EVENT0 is what is used in other methods here. Should this be different?

Reply:
Caveat - replying directly; I haven't reviewed the rest of this file to check every case.

`TRACE_EVENT0` emits two events: a begin immediately followed by an end.
`TRACE_EVENT_INSTANT0` just logs an instant event, which is probably what we want in this case.

I'll take a scan through the rest and we can update them if necessary.

Reply:
Isn't TRACE_EVENT0 essentially used when we want to capture time it took for particular action to complete? Why wouldn't we want to get that duration for this event?

Reply:
If you do, then `TRACE_EVENT0` is what you want.